doc/nrf/ug_bt_fast_pair.rst

```diff
@@ -97,6 +97,15 @@ The Bluetooth privacy is selected by the Fast Pair service, but you must make su
 
 See the official `Fast Pair Advertising`_ documentation for detailed information about the requirements related to discoverable and not discoverable advertising.
 
+Battery Notification extension
+------------------------------
+
+The `Fast Pair`_ documentation specifies `Fast Pair Battery Notification extension`_.
```

Comment:
I think instead of starting with link to the fast fair documentation, we could add some details about the use of 
"Battery Notification extension" in NCS (introduction statement) and after the explanation, we could provide the link to the Fast pair documentation as we are doing for the other sections.